instrumentation/active_job/lib/opentelemetry/instrumentation/active_job/patches/active_job_callbacks.rb

```diff
@@ -81,6 +86,10 @@ def job_attributes(job)
             otel_attributes.compact
           end
 
+          def milliseconds_since(timestamp)
```

Comment:
I'm a little concerned about "polluting" the ActiveJob method namespace with this method, so I'm open to alternatives.  It would also seem okay to me to inline this logic within the `job_attributes` method 🤷‍♂️